src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populations = [...populationList];
+
+let total = 0;
+
+populations.forEach((population) => {
```

Comment:
```suggestion
[...populationsList].forEach((population) => {
```
and make an array directly here

not a mistake though)